src/sonic-yang-models/yang-models/sonic-nat.yang

```diff
@@ -0,0 +1,316 @@
+                }   
+                
+                leaf ip_protocol {
+                    description "IP Protocol (tcp or udp) for the Static NAPT entry.";
+
+                    type stypes:ip-protocol-type;
+                }   
+                
+                leaf global_l4_port {
+                    description "Global L4 port for the Static NAPT entry.";
+
+                    type inet:port-number;
+                }   
+
+                leaf local_ip {
+                    description "Local ip for the Static NAPT entry.";
+
+                    mandatory true;
+                    type inet:ipv4-address;
+                }   
+
+                leaf local_port {
+                    description "Local port for the Static NAPT entry.";
+
+                    mandatory true;
+                    type inet:port-number ;
+                }
+                
+                leaf nat_type {
+                    description "Nat type for the static napt entry - snat or dnat";
+
+                    type nat-type;
+                    default dnat;
+                }   
+                
+                leaf twice_nat_id {
+                    description "Twice nat id for the static napt to achieve the twice napt";
+
+                    type uint16 { 
+                        range "1..9999" {
+                            error-message "Invalid twice nat id for the static NAPT.";
+                            error-app-tag twice-nat-id-invalid;
+                        }   
+                    }   
+                }
+            }
+        }
+
+        container STATIC_NAT {
+
+            description "STATIC_NAT table part of config_db.json";
+
+            list STATIC_NAT_LIST {
+                key "global_ip";
+
+                leaf global_ip {
+                    description "Global ip for the Static NAT entry.";
+
+                    type inet:ipv4-address;
+                }
+
+                leaf local_ip {
+                    description "Local ip for the Static NAT entry.";
+
+                    mandatory true;
+                    type inet:ipv4-address;
+                }
+
+                leaf nat_type {
+                    description "Nat type for the static nat entry - snat or dnat";
+
+                    type nat-type;
+                    default dnat;
+                }
+
+                leaf twice_nat_id {
+                    description "Twice nat id for the static nat to achieve the twice nat";
+
+                    type uint16 { 
+                        range "1..9999" {
+                            error-message "Invalid twice nat id for the static NAT.";
+                            error-app-tag twice-nat-id-invalid; 
+                        }   
+                    }   
+                }
+            }
+        }
+
+        container NAT_GLOBAL {
+
+            description "NAT_GLOBAL table part of config_db.json";
+
+            list NAT_GLOBAL_LIST {
```

Comment:
Replace list (with max-elements 1) with a container? e.g container NAT_GLOBAL { container values {...}}

Reply:
Seems like List is mandatory here, seeing errors when I replace list with container.
Is it supported ? If yes, could you please give me some reference here.

Reply:
Please refer  TACPLUS container in file  src/sonic-yang-models/yang-models/sonic-system-tacacs.yang 
PR - https://github.com/Azure/sonic-buildimage/pull/7671/files

Reply:
Done, replaced list with a container.